components/core/src/event.rs

```diff
@@ -262,9 +260,9 @@ pub struct EventLogger {
 }
 
 impl EventLogger {
-    pub fn new(service_name: &str, enabled: bool) -> Self {
+    pub fn new<T: Into<PathBuf>>(log_dir: T, enabled: bool) -> Self {
         EventLogger {
-            log_dir: svc_var_path(&service_name),
+            log_dir: log_dir.into(),
```

Comment:
Is this logging to a file rather than stdout because we want to collect the data for some other reason? Services in general should log to stdout.